### PR TITLE
Af exp br

### PR DESCRIPTION
**Pull request by starkaf**

### CODE REVIEW

File: backend/CMakeLists.txt [hunk 1]
```diff
@@ -30,7 +30,8 @@ set(TFL_BUILD_DIR ${TF_ENV}/tflite_build)
 set(TFL_LIB_PATH /usr/lib/libtensorflowlite.so)
 set(TFL_GPU_LIB_PATH /usr/lib/libtensorflowlite_gpu_delegate.so)
 
-set(LIB_PATH /usr/lib/x86_64-linux-gnu)
+#set(LIB_PATH /usr/lib/x86_64-linux-gnu)
+set(LIB_PATH /usr/lib/aarch64-linux-gnu)
```
Comment (alxhoff) [inline backend/CMakeLists.txt]: is this something for the dev board?

File: utils/usb/__init__.py
Comment (alxhoff) [inline utils/usb/__init__.py:1]: same question as with the last PR, does usbmon change and we should pass it in the the make system as an argument?

File: utils/usb/debug.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def capture_stream(signalsQ:Queue, dataQ:Queue, timeout:int, l:Log) -> None:
 
     packets = []
     context = StreamContext()
-    capture = pyshark.LiveCapture(interface='usbmon4', display_filter=get_filter(addr))
+    capture = pyshark.LiveCapture(interface='usbmon2', display_filter=get_filter(addr))
```
Comment (alxhoff) [inline utils/usb/debug.py:81]: same here

File: utils/usb/usb.py
Comment (alxhoff) [inline utils/usb/usb.py:1]: and again

File: backend/CMakeLists.txt (hunk 1 above)
Comment (starkaf) [reply backend/CMakeLists.txt]: yes, will probably be changed to depend on an external argument we provide to the Makefile to define what platform we're dealing with before building the backend

File: utils/usb/__init__.py
Comment (starkaf) [reply utils/usb/__init__.py:1]: yes, same issue, should I take care of the change to make it be passed as an arg?

File: backend/CMakeLists.txt (hunk 1 above)
Comment (alxhoff) [reply backend/CMakeLists.txt]: yep, then we should handle this via an arg. Will set it back to the normal docker one for now then

File: utils/usb/__init__.py
Comment (alxhoff) [reply utils/usb/__init__.py:1]: Sounds good!

Comment (alxhoff) [reply utils/usb/__init__.py:1]: i'll merge this now and we can add it in later

**Pull request merged**